tools/local-compliance/scripts/local-compliance.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -exEuo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+
+# Use less standart local port 8554
+kubectl -n stackrox port-forward deploy/sensor 8554:8443 > /dev/null &
+PID=$!
+
+function ctrl_c() {
+    echo "Killing PID $PID"
+    rm -f "${DIR}/../certs/ca.pem" "${DIR}/../certs/collector-cert.pem" "${DIR}/../certs/collector-key.pem"
+    kill $PID
+}
+
+trap ctrl_c INT EXIT
+
+mkdir -p "${DIR}/../certs"
+
+kubectl -n stackrox get secrets collector-tls -ojsonpath='{.data.ca\.pem}' \
+    | base64 -D -o "${DIR}/../certs/ca.pem"
+kubectl -n stackrox get secrets collector-tls -ojsonpath='{.data.collector-cert\.pem}' \
+    | base64 -D -o "${DIR}/../certs/collector-cert.pem"
+kubectl -n stackrox get secrets collector-tls -ojsonpath='{.data.collector-key\.pem}' \
+    | base64 -D -o "${DIR}/../certs/collector-key.pem"
+
+ROX_NODE_SCANNING_MAX_INITIAL_WAIT="1s" \
+ROX_NODE_SCANNING_INTERVAL="1s" \
```

Comment:
I think it would be good to have these two configurable.

Reply:
You mean configurable as params of the shell script?

Reply:
Yes!

Reply:
Let me address that in a follow-up if that is fine. I would like to get this merged to unlock the work on the last phase of [ROX-15458](https://issues.redhat.com/browse/ROX-15458)